### PR TITLE
Create new UI

### PR DESCRIPTION
**Pull request by chunshou-Liu**

### CODE REVIEW

File: ui/all.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+$(document).ready(function(){
```
Comment (Pandede) [inline ui/all.js:1]: Add comments for simple explanation

File: ui/all.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    // console.log(grid.getAttribute("row", "col"))
+    $('.keyboard').on('click', 'button', function(event) {
+        $(this).parent().parent().find('button').removeClass('activate');
+        $(this).addClass('activate');
+        
+    });
+
+    $('.keyboard').on('click', '.btn', function(event) {
+        if (pointer[1] >= 5) {
+            alert("please press enter or delete!");
```
Comment (Pandede) [inline ui/all.js:29]: May delete the alert part

Comment (chunshou-Liu) [reply ui/all.js:29]: would delete that after finishing the hint animations

**Review by Pandede — APPROVED** [review]
LGFM!

**Pull request merged**